### ISSUE TITLE
Meeting 26 April

### ISSUE THREAD

**Issue by wwjvdsande** [OpenSourceMycetoma/Series-1-Fenarimols#74]
Open Source Mycetoma (MycetOS) Meeting
Recurring Meeting Zoom link: https://uni-sydney.zoom.us/j/99281549497

Time: New York 16:00 - UTC 20:00 - London 21:00 - Rotterdam 22:00 - Sydney 06:00

Chair: ErasmusMC

Minute taker: UCL

Recording can be found [here]

Minutes can found below

Link to previous meeting https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/73

Agenda

**Comment by mattodd** [comment]
Time is fine for me.
For the paper, am I right in thinking that the Google Doc is now out of date and that the current version of the paper is the one linked [here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/55#issuecomment-1087216694)? And how about all the various files needed for the paper - where are they collected? It would be useful to be 100% sure on the location of both, and how we're going to go about editing them together, to make sure there is never anyone working on outdated versions. My preference would be for Google Docs to the very last moment, and to use Github as the shared filestore.

**Comment by wwjvdsande** [comment]
@mattodd @OpenSourceMycetoma/corecontrib the most recent paper is in dropbox. There were issues with the chemical figures in the Google doc document and that was why we all decided to use the dropbox.

**Comment by MA-Jjingyi** [comment]
Hello, everyone. the data are updated in the excel. and I share the slides here too. several experiments are on going, and I will update them as soon as possible.
[26042022 MyctOS meeting -MA.pptx](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/8580968/26042022.MyctOS.meeting.-MA.pptx)
.

**Comment by wwjvdsande** [comment]
@bebi78 that is a good suggestion

**Comment by kym834** [comment]
Hi all, just updating that @dmitrij176, @fantasy121 and I have had a meeting to discuss the experimental write up etc with a few things still needed to be decided upon, see #75.

@Wilson-Lm, @wwjvdsande @mattodd - I'm wondering if for the paper draft, do you want to have all the chemistry experimental write up included? This is not always included in the paper itself but rather as Supp Info for many med chem papers. If not, I think we could very quickly get the draft complete as much of what still needs to be done from our end is formatting and writing up all the experimental. Let me know what your thoughts are here.

**Comment by bebi78** [comment]
@kym834 It also depends on the journal and you have to check the author guidelines of the chosen journal. 
For J. Med. Chem. it is usually sufficient to provide typical procedures for selected compounds in the experimental section of the main manuscript, the rest can be placed in the supplement (see attached file). But other Med Chem journals want to have the procedures and analytics of all test compounds in the main manuscript.

[jm061379o.pdf](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/8619963/jm061379o.pdf)

**Comment by mattodd** [comment]
Hi all - yes, the extent to which the data will go in the paper will vary, but so long as it's prepared in one file, we can cut and paste. Certainly any known molecules would be expected to be in the SI, and if we're using previous procedures we can just reference those without needing to write them out again.

**Comment by Wilson-Lm** [comment]
> I was just throwing out there a possibility that would allow for the draft @Wilson-Lm needs to submit for his PhD to be complete sooner rather than later. Given we haven't chosen a journal yet and that (I'll make an assumption here) assesment will be on the work @Wilson-Lm has done rather than the over all paper, might be sufficient and a reasonable enough argument that would allow him to submit and not have to delay any further. Of course then in future iteration toward the final manuscript we can cut and paste as Mat suggests what is needed dependant on the journal.
> 
> I'm not 100% sure if this would be suitable for PhD by publication but just some food for thought.

Hi all, indeed what @kym834 mentioned, it is not necessary to have all the chemistry experimental write up included in my thesis. In my case, focus would be mainly placed on the biological evaluation and the concept of drug repurposing. In the Netherlands, our thesis is a collection of articles published by the PhD candidate + an intro and discussion chapters. It is allowed to have a couple of 'articles' in there which are not complete - they can be either in submission or in the case of this fenarimol article - a concept.

**Comment by kym834** [comment]
@Wilson-Lm OK. We can prioritise pulling together all the data and experimental for those tested in vivo first so you can include that in your thesis then finish off all the other compounds.

**Comment by kym834** [comment]
I've added a new sheet to the Master List workbook called 'MYOS in vivo' where I've just pulled out those that have been tested in vivo (we can delete this once we've sorted out if need be). @Wilson-Lm can you double check that this has all the compounds that have been tested in vivo so that we don't miss any?

**Comment by bebi78** [comment]
@mattodd @fantasy121 I think one contributor asked for the mechanism of action of the new fenarimols during the meeting. Maybe the pyridine ring of one of the triarylmethanol derivatives can be alkylated with methyl iodide or benzyl bromide to prevent iron binding at the assumed cytochrome target. If the alkyl pyridinium derivative shows no antifungal activity anymore this may be a hint at cytochrome iron binding as mechanism of action (as expected). 
Well, if the pyridinium is still active you have an entry to a new series of antifungal compounds (alkyl pyridinium compounds) with a likely different mode of action.

**Comment by dmitrij176** [comment]
> @mattodd @fantasy121 I think one contributor asked for the mechanism of action of the new fenarimols during the meeting. Maybe the pyridine ring of one of the triarylmethanol derivatives can be alkylated with methyl iodide or benzyl bromide to prevent iron binding at the assumed cytochrome target. If the alkyl pyridinium derivative shows no antifungal activity anymore this may be a hint at cytochrome iron binding as mechanism of action (as expected).
> Well, if the pyridinium is still active you have an entry to a new series of antifungal compounds (alkyl pyridinium compounds) with a likely different mode of action.

@bebi78 at least that what I would expect to happen as well. This is based on the known role of pyridine/azole moieties as iron binding regions. Maybe it would be useful to get at least several molecules for proof of concept studies. In addition, the importance of N-containing ring will be also evaluated in the next batch with deoxybenzoin and deoxyanisoin scaffolds that Bernard provided recently.  The two ketoximes have already been syntesised and have pyrifenox-like (methyl) varible group R.

**Comment by bebi78** [comment]
@dmitrij176 Yes, the benzoin and anisoin derived ketoximes can also give hints at the role of the pyridine ring. One-two compounds should be sufficient for first results. I suggested the triarylmethanol fenarimols because some of them have only one pyridine ring and no additional N-heterocycles such as piperazines, which might also undergo alkylation and might lead to some sort of false results based on the alkylation of the piperazine.
